miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/BulkDilutionIT.java

```diff
@@ -0,0 +1,225 @@
+
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
+
+import uk.ac.bbsrc.tgac.miso.core.data.impl.LibraryDilution;
+import uk.ac.bbsrc.tgac.miso.webapp.integrationtest.page.BulkDilutionPage;
+import uk.ac.bbsrc.tgac.miso.webapp.integrationtest.page.BulkDilutionPage.Columns;
+import uk.ac.bbsrc.tgac.miso.webapp.integrationtest.page.element.HandsOnTable;
+import uk.ac.bbsrc.tgac.miso.webapp.integrationtest.page.element.HandsOnTableSaveResult;
+
+public class BulkDilutionIT extends AbstractIT {
+
+  private static final Logger log = LoggerFactory.getLogger(BulkDilutionIT.class);
+
+  private static final Set<String> columns = Sets.newHashSet(Columns.NAME, Columns.ID_BARCODE, Columns.LIBRARY_ALIAS,
+      Columns.CONCENTRATION, Columns.CREATION_DATE, Columns.TARGETED_SEQUENCING);
+
+  private static final String NO_TAR_SEQ = "(None)";
+
+  @Before
+  public void setup() {
+    loginAdmin();
+  }
+
+  @Test
+  public void testEditSetup() throws Exception {
+    // Goal: ensure all expected fields are present and no extra
+    BulkDilutionPage page = BulkDilutionPage.getForEdit(getDriver(), getBaseUrl(), Sets.newHashSet(304L, 305L));
+    HandsOnTable table = page.getTable();
+    List<String> headings = table.getColumnHeadings();
+    assertEquals(columns.size(), headings.size());
+    for (String col : columns) {
+      assertTrue("Check for column: '" + col + "'", headings.contains(col));
+    }
+    assertEquals(2, table.getRowCount());
+  }
+
+  @Test
+  public void testPropagateSetup() throws Exception {
+    // Goal: ensure all expected fields are present and no extra
+    BulkDilutionPage page = BulkDilutionPage.getForPropagate(getDriver(), getBaseUrl(),
+        Sets.newHashSet(304L));
+    HandsOnTable table = page.getTable();
+    List<String> headings = table.getColumnHeadings();
+    assertEquals(columns.size(), headings.size());
+    for (String col : columns) {
+      assertTrue("Check for column: '" + col + "'", headings.contains(col));
+    }
+    assertEquals(1, table.getRowCount());
+
+    assertEquals("DILT_0001_nn_n_PE_304_WG", table.getText(Columns.LIBRARY_ALIAS, 0));
+  }
+
+  @Test
+  public void testCreateDropdowns() throws Exception {
+    BulkDilutionPage page = BulkDilutionPage.getForPropagate(getDriver(), getBaseUrl(),
+        Sets.newHashSet(304L));
+    HandsOnTable table = page.getTable();
+    
+    List<String> targetedSequencings = table.getDropdownOptions(Columns.TARGETED_SEQUENCING, 0);
+    assertTrue(targetedSequencings.contains("Test TarSeq One"));
+    assertTrue(targetedSequencings.contains("Test TarSeq Two"));
+    assertTrue(targetedSequencings.contains("(None)"));
+    assertEquals("targeted sequencing size", 3, targetedSequencings.size()); // two plus "(None)"
+  }
+
+  @Test
+  public void testCreateDependencyCells() throws Exception {
+    // none at this time
```

Comment:
Remove empty test methods as they'll add to the growing build time (WebDriver initialisation and login will still happen even if the method is empty)